tests/fixtures/scorecard.json

```diff
@@ -0,0 +1,333 @@
+{
```

Comment:
This file isn't strictly necessary, as there is already a scorecard fixture, but it has insertions and the interaction of transforms on a scorecard is even less well defined than without them.